### PR TITLE
Adding a demo robot

### PR DESCRIPTION
**Pull request by orduno**
... for testing the nav stack. We can use this one while we wait for Turtlebot to be fully supported in ROS2.

This robot has a differential drive, laser, and imu sensors. Uses the corresponding ros_gazebo_plugins to interface with Gazebo.

### CODE REVIEW

**Comment by SteveMacenski** [comment]
Is this here because the `spawn_model` hasnt been ported to ros2 yet (unsure)? If it has, there are world files turtlebot simulation (that's already a ros1 kinetic/melodic dependency nav2 pulls in). I'd just grab one of those and a model spawner in a (py) launch file 

https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/master/turtlebot3_gazebo/worlds

File: nav2_bringup/launch/gazebo_ros_simplebot.world [hunk 1]
```diff
@@ -0,0 +1,410 @@
+<?xml version="1.0"?>
+<!--
```
Comment (crdelsey) [inline nav2_bringup/launch/gazebo_ros_simplebot.world]: Does it make more sense to put this documentation in a README file in the directory? I wouldn't normally think to look inside a .world file for instructions.

Comment (mkhansen-intel) [reply nav2_bringup/launch/gazebo_ros_simplebot.world]: I agree with Carl, can you add a short README instead?

**Comment by orduno** [comment]
> Is this here because the `spawn_model` hasnt been ported to ros2 yet (unsure)? If it has, there are world files turtlebot simulation (that's already a ros1 kinetic/melodic dependency nav2 pulls in). I'd just grab one of those and a model spawner in a (py) launch file
> 
> https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/master/turtlebot3_gazebo/worlds

Yes, the `spawn_model` hasn't been ported to ROS2.

File: nav2_bringup/launch/gazebo_ros_simplebot.world (hunk 1 above)
Comment (orduno) [reply nav2_bringup/launch/gazebo_ros_simplebot.world]: Agree, the README file is still work in progress.

File: nav2_bringup/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      * /ray/pointcloud
+      * /ray/laserscan
+
+  *Note that the laser scans are not working on RViz due to https://github.com/ros2/rviz/issues/332*
+
+  Provide a transform for the laser sensor:
+
+      $ ros2 run tf2_ros static_transform_publisher 0.75 0 0.484 0 0 0 base_link ray_link
+
+  On RViz, change the Fixed Frame" to ```ray_link```
```
Comment (mjeronimo) [inline nav2_bringup/README.md]: Stray quote (") after Fixed Frame?

File: nav2_bringup/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Bringup
+
+## Before starting:
+- Build ROS2 from master branch
```
Comment (mjeronimo) [inline nav2_bringup/README.md]: "from the master branch"

File: nav2_bringup/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- If working with Ubuntu 16, install [Gazebo 9](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
+  - In step #3:
```
Comment (mjeronimo) [inline nav2_bringup/README.md]: It was immediately obvious that the text was referring to step 3 n the gazebo install tutorial. Could make this explicit: "In step #3 in the gazebo installation instructions" or something like that.

File: nav2_bringup/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# Bringup
+
+## Before starting:
```
Comment (mjeronimo) [inline nav2_bringup/README.md]: I'm not clear if this is a README for the bring-up of the Navigation 2 system or just describing how to bring up gazebo. The location in nav2_bringup would indicate that it should describe how to bring up the whole system. We should also include instructions for starting the nav2 system.

**Comment by mkhansen-intel** [comment]
Closing this one at least until Carlos is back, at which time we can revisit if needed.

**Pull request closed without merging**